backend/spec/views/spree/admin/shared/navigation_footer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe "spree/admin/shared/_navigation_footer", type: :view do
+  let(:user) { create(:admin_user) }
```

Comment:
Theoretically, even `Spree::User.new` should do fine here

Reply:
Yes, `new` is faster. `FactoryGirl.build` is marginally slower, as it will persist associated objects. For now though I think it's fine, let's deal with this in a subsequent PR. The answer to the conundrum would probably `FactoryGirl.build_stubbed`.